src/function/mod.rs

```diff
@@ -390,36 +429,42 @@ impl Function {
             return result;
         }
         let out_ty = &self.schema.output;
-        self.nodes.insert_and_merge(scratch, timestamp, |prev| {
-            if let Some(mut prev) = prev {
-                out_ty.canonicalize(&mut prev, uf);
-                let mut appended = false;
-                if self.merge.on_merge.is_some() && prev != out_val {
-                    deferred_merges.push((scratch.clone(), prev, out_val));
-                    appended = true;
-                }
-                match &self.merge.merge_vals {
-                    MergeFn::Union => {
-                        debug_assert!(self.schema.output.is_eq_sort());
-                        uf.union_values(prev, out_val, self.schema.output.name())
+        self.nodes.insert_and_merge(
+            scratch,
+            timestamp,
+            |prev| {
```

Comment:
This looks like a big change, but I think the function remained the same. The formatted just changed the indentation of the function when I added another arg.